src/DynamoCoreWpf/Views/Search/LibrarySearchView.xaml

```diff
@@ -121,8 +121,7 @@
                 <Setter Property="Template">
                     <Setter.Value>
                         <ControlTemplate TargetType="{x:Type TreeViewItem}">
-                            <Button Command="{Binding ClickedCommand}"
-                                    Focusable="False"
```

Comment:
@aosyatnik - I think that is ok. Otherwise the button will never get focus, and it scrolls all the way down. Is there a better way to do that?